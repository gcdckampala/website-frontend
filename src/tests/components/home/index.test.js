import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import Home from '../../../components/home';

describe('Provider and Home', () => {
  const home = shallow(<Home />);

  it('renders <Provider/> correctly', () => {
    expect(home).toMatchSnapshot();
  });
});
