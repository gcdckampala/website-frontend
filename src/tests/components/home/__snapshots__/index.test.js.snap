// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and App renders <Provider/> correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</Router>
`;

exports[`Provider and Home renders <Provider/> correctly 1`] = `
<div>
  <h1>
    Welcome To GCDC 
  </h1>
</div>
`;
